docs/source/guides/importpilotmodel.rst

```diff
@@ -0,0 +1,236 @@
+.. _importpilotmodel:
+
+Pilot Model Importing
+=====
+
+Tools:
+^^^^^
+
+.. _Blender: https://www.blender.org/
+.. _Blender Source Tools: http://steamreview.org/BlenderSourceTools/
+.. _Titanfall VPK Tools: https://retryy.gitbook.io/tf2/Wiki/Tools/vpk-tools
+.. _Crowbar: https://steamcommunity.com/groups/CrowbarTool
+.. _Legion+: https://github.com/r-ex/LegionPlus
+.. _RePak: https://github.com/r-ex/RePak
+.. _mdlshit: https://github.com/headassbtw/mdlshit-binaries
+.. _Text Editors: https://retryy.gitbook.io/tf2/Wiki/Tools/general-pc-tools#text-editing
+
+Download each tool. 
+
+`Text Editors`_ - Just a text editor
+
+`Titanfall VPK Tools`_ - VPK viewer, extractor, and repacker.
+   "Titanfall VPK Tool" is the vpk tool used. Made by cra0.
+
+`Crowbar`_ - Decompile, recompile, and a ton of other model and source asset stuff
+
+`Blender`_ - 2D/3D Workspace
+
+`Blender Source Tools`_ - Source Tools plugin for Blender
+
+`Legion+`_  - Texture, audio, shaderset, and material exporter
+   "Normal" Legion doesn't work
+
+`RePak`_ - Create Apex and Titanfall 2 .rpak for textures
+
+`mdlshit`_ - Magically convert .mdl to "titanfall 2" .mdl 
+
+Exporting the pilot model from the vpk
+^^^^^
+
+Locate your Titanfall 2 game folder
+-----
+
+For Origin you should find it in this directory ``Origin Games\Titanfall2``.
```

Comment:
Given that we also have EA App now, it might be better to just link to a resource that tells you how to find the game location.

Or otherwise at least describe how to do it for EA App as well now ^^

Also keep in mind that players can install in different locations so it might make more sense to tell them to go via e.g. Steam -> game properties -> local files -> open location instead of giving them a path,